iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -23,47 +30,75 @@ public enum ErrorCode
         BulkRegistryOperationFailure = 400013,
         CannotRegisterModuleToModule = 400301,
 
-        //Unauthorized - 401
+        /// <summary>
+        /// Unauthorized - 401
+        /// </summary>
         IotHubNotFound = 401001,
+
         IotHubUnauthorizedAccess = 401002,
         IotHubUnauthorized = 401003,
```

Comment:
Again, the naming of this enum isn't doing itself any favors. It could use a comment about "This error code signals that the cloud to device operation failed because the device is not currently connected to IoT Hub"

https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-troubleshoot-error-404103-devicenotonline

Reply:
Thanks for the link. It has details on a lot of errors. I will include all the useful ones and update my PR.